.travis.yml

```diff
@@ -20,5 +20,6 @@ before_install:
   - travis_retry composer self-update
 
 install:
+  - if [[ "$TRAVIS_PHP_VERSION" != "5.3" ]] && [[ "$TRAVIS_PHP_VERSION" != "5.4" ]]; then composer require symfony/psr-http-message-bridge:dev-wip zendframework/zend-diactoros; fi;
```

Comment:
`--dev` is only implicit with the `install` and `update` commands. By default, the dependency would be added to the `require` section, but not to the `require-dev` section. This shouldn't make any difference here I guess, but technically it's a dev dependency.